src/components/NewMessageModal.vue

```diff
@@ -23,19 +24,17 @@ import logger from '../logger'
 import { toPlain } from '../util/text'
 import { saveDraft } from '../service/MessageService'
 import Composer from './Composer'
+import { showError, showUndo } from '@nextcloud/dialogs'
 import { translate as t } from '@nextcloud/l10n'
 
+const UNDO_DELAY = 7 * 1000
```

Comment:
TODO: could be exposed from the capabilities
Maybe a followup for cleaner code :)

Reply:
what capabilities?

Reply:
That UNDO is supported and the timeout which the backend will use before sending the message?
Or is it being sen straight from the front only?

Reply:
Straight outta frontend. The backend only sends the message if the tab/browser is closed

Reply:
Nevertheless I forgot to add the delay to the message's `sendAt`, so the backend could have picked up the message before the elapsed undo timer :see_no_evil: 

Fixed.